docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +107,261 @@ To manage file storages:
 
-3. To delete a file, use the `deleteFile()` method:
+   The directory is created, in destination pointed by `path` and successCallback function should be called. If directory can not be created for some reason, the errorCallback is called and error message can be logged.
 
    ```
-   function onDelete() {
-       console.log('deletedFile() is successfully done.');
+   var newPath = 'documents/subDir'
+   var successCallback = function(newPath) {
+       console.log('New directory has been created: ' + newPath);
    }
-
-   documentsDir.deleteFile(newFile.fullPath, onDelete);
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.createDirectory('documents/newDir',successCallback, errorCallback);
    ```
 
-4. To delete a directory, use the `deleteDirectory()` method.
+2. To delete a directory, use the `deleteDirectory()` method.
 
    The second parameter defines whether the deletion is performed recursively for the sub-directories as well. If the parameter is set to `false`, the directory is deleted only if it is empty.
 
    ```
-   documentsDir.deleteDirectory(newDir.fullPath, false, onDelete);
-   anotherNewDir.parent.deleteDirectory(anotherNewDir.fullPath, false, onDelete);
+   var fullPath = 'documents/subDir'
+   var callback = function(modifiedDirectory) {
+       console.log('deleteDirectory() is successfully done. Modified (parent) directory: ' + modifiedDirectory);
+   }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.deleteDirectory(fullPath, true, callback, errorCallback);
    ```
 
-## Retrieving Files and File Details
+## Deleting Files
 
-You can retrieve a list of files or file URIs using the `listFiles()` and `toURI()` methods. The URI can be used to identify the file, for example, by using it as the `src` attribute on an HTML `img` element.
+You can create directories using the `createDirectory()` method. The directory is created at `path` specified in parameter.
+You can delete files using `deleteFile()` method. Target pointed by `path` specified will be deleted from filesystem.
 
-You can retrieve file content as a `DOMString` with the `readAsText()` method. The encoding input parameter of the method defines the format in which the file content is returned.
-
-To get files and file details from the file system:
-
-1. To access a specific file or directory within the file system, retrieve a file handle using the `resolve()` method of the `FileSystemManager` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#FileSystemManager), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#FileSystemManager), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#FileSystemManager) applications):
+1. To delete a file, use the `deleteFile()` method:
```

Comment:
Please change the ordered list to unordered list (bullets) as it is not a sequential or a procedural step.

Reply:
Done.